docs/painless/painless-contexts/painless-watcher-context-variables.asciidoc

```diff
@@ -30,10 +30,3 @@ The following variables are available in all watcher contexts.
 `ctx['payload']` (`Map`, read-only)::
         The accessible watch data based upon the
         {ref}/input.html[watch input].
-
```

Comment:
Why remove this?

Reply:
Because this section was being reused in other pages, we ended up with two *API* sections on those pages. For example, the [Watcher condition context](https://www.elastic.co/guide/en/elasticsearch/painless/7.11/painless-watcher-condition-context.html) page looks like this currently:

![image](https://user-images.githubusercontent.com/25848033/106051301-46055100-60b6-11eb-8eeb-8c7eb2f0aa83.png)